dist/drupal-conventions.yml

```diff
@@ -1,15 +1,16 @@
 imports:
-  - { resource: base-conventions.yml }
+    - { resource: base-conventions.yml }
 
 parameters:
-  tasks.phpcs.standard: ./vendor/drupal/coder/coder_sniffer/Drupal
-  tasks.phpcs.triggered_by:
-    - php
-    - module
-    - inc
-    - theme
-  tasks.phpmd.triggered_by:
-    - php
-    - module
-    - inc
-    - theme
+    tasks.phpcs.standard: ./vendor/drupal/coder/coder_sniffer/Drupal
+    tasks.phpcs.triggered_by:
```

Comment:
We should add these 3 following extensions as `phpcsfixer`
-`install`
-`module`
-`profile`

Reply:
Fixed.